playbooks/openstack-osh_instance.yml

```diff
@@ -0,0 +1,89 @@
+---
+- hosts: localhost
+  gather_facts: no
+  connection: local
+  roles:
+    - ansible-config_template
+  post_tasks:
+    - name: Load generic vars
+      include_vars: "{{ playbook_dir }}/../vars/common-vars.yml"
+    - name: Load openstack vars
+      include_vars: "{{ playbook_dir }}/../vars/deploy-on-openstack.yml"
+
+    - name: Handle creation
+      when:
+        - not ((osh_node_delete | default('false'))| bool)
+      block:
```

Comment:
Same as step one, no rescue block may lead to improper cleanup of resources

Reply:
Agreed.